pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -40,8 +40,8 @@
     "System.Threading.Thread": "4.0.0",
     "System.Threading.ThreadPool": "4.0.10",
     "Libuv": "1.9.1",
-    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24703-01",
```

Comment:
It should be able to rebase as is, I had to do this to get past the package issues.